build_tools/docker/context/setup_mmperf.sh

```diff
@@ -24,11 +24,13 @@ export REPO_DIR=$1
 export REPO_SHA=$2
 
 pushd ${REPO_DIR}
-git clone --jobs 8 --depth 1 --no-single-branch --recurse-submodules https://github.com/mmperf/mmperf.git
-pushd mmperf
 
-# Checkout a specific commit.
+mkdir mmperf
+pushd mmperf
+git init
+git fetch --depth 1 https://github.com/mmperf/mmperf.git "${REPO_SHA}"
 git checkout ${REPO_SHA}
+git submodule update --init --recursive --jobs 8 --depth 1
```

Comment:
IIUC this is the part we try to fix in this PR. Can you briefly explain in the PR description? Just to help understanding.

Reply:
+1. Please add more detail to the PR description. You can link the previous PR and my comment as well

Reply:
Added